bom/pom.xml

```diff
@@ -16,6 +16,7 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
+<!-- Portions Copyright [2018] [Payara Foundation and/or its affiliates] -->
```

Comment:
Please use the portions copyright inside the copyright comment:
```
/**
 * Copyright Oracle...
 * (Portions) Copyright Payara
 *  ...
```
The new glassfish copyrigh plugin v 1.49 should handle it

Reply:
Hi @jansupol, Here #3925, you commented to remove all portion copyright but In this PR, you suggested to use the below template. Should I remove the all portion copyright in this PR too ?

> ```
> /**
>  * Copyright Oracle...
>  * (Portions) Copyright Payara
>  *  ...

Reply:
Hi @jGauravGupta, first of all, once again I am sorry for suggesting the "(Portions)". I must have seen it somewhere, but the string makes the GF copyright plugin fail. 
We have discussed this already and you have removed the string from this PR, observing the changes, the string is not there.